core/state/statedb.go

```diff
@@ -1341,6 +1341,10 @@ func (s *StateDB) Prepare(rules params.Rules, sender, coinbase common.Address, d
 		al.AddAddress(sender)
 		if dst != nil {
 			al.AddAddress(*dst)
+			// If the dst has a delegation, also warm its target.
+			if addr, ok := types.ParseDelegation(s.GetCode(*dst)); ok {
```

Comment:
Doesn't _have_ to be, really. It's gated on the prefix `0xef, 0x01`. No contract has that prefix, so this will only ever return `common.Address{}, false` and be a no-op